client/context/helpers.go

```diff
@@ -126,7 +126,7 @@ func (ctx CoreContext) GetFromAddress() (from sdk.Address, err error) {
 		return nil, errors.Errorf("no key for: %s", name)
 	}
 
-	return info.GetPubKey().Address(), nil
+	return sdk.AccAddress(info.GetPubKey().Address()), nil
```

Comment:
Would it make sense to have a helper method to create an AccAddress from pubkey? I think calling it once you already have a thing called an address is a bit confusing.

Reply:
I think casting it from Address to AccAddress make sense.  Also, I do need to turn crypto.PubKey into sdk.AccPubKeys everywhere (in a separate PR).  So I'll have helper function to turn AccPubKey to AccAddress though.